app/policies/fund_policy.rb

```diff
@@ -4,19 +4,23 @@ def index?
   end
 
   def show?
-    true
+    user.administrator? || user.organisations.include?(record.organisation)
   end
 
   def create?
-    true
+    user.administrator? || associated_fund_manager?(user: user, record: record)
   end
 
   def update?
-    true
+    user.administrator? || associated_fund_manager?(user: user, record: record)
   end
 
   def destroy?
-    true
+    user.administrator? || associated_fund_manager?(user: user, record: record)
+  end
+
+  private def associated_fund_manager?(user:, record:)
```

Comment:
Since `user` and `record` are globally available in the policy class, they don't need to be passed as parameters.

So you could simplify this to `user.administrator? || associated_fund_manager?`

Reply:
That'd be much better! 💥